workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/executor/message/SendMessageExecutor.java

```diff
@@ -105,6 +111,27 @@ private Message buildMessage(ActivityExecutorContext<SendMessage> execution) thr
     return builder.build();
   }
 
+  static String extractContent(Object contentObj, Map<String, Object> parameters, MessageService messageService)
+      throws IOException {
+    String content;
+    try {
+      String template = OBJECT_MAPPER.convertValue(contentObj, SendMessage.Content.class)
+          .getTemplate();
+      Map<String, String> params = parameters.entrySet().stream()
+          .collect(Collectors.toMap(Map.Entry::getKey, e -> e.getValue().toString()));
+      File file = new File(template);
+      if (file.getName().endsWith(".ftl")) {
```

Comment:
can't we consider all the files to be templates? would that break something?

Reply:
yes, for some reason I tested it before and it was not working. But now it is, so probably I had a different error. 
We can use directly the messageService template from the bdk in both cases.